pallets/claims/src/tests.rs

```diff
@@ -17,14 +17,24 @@
 
 use crate::mock::*;
 use crate::{
-	Claims, EcdsaSignature, Error, EthereumAddress, InvalidTransaction, SignedExtension, ValidTransaction,
-	ValidateClaim,
+	Claims, Config, EcdsaSignature, Error, EthereumAddress, InvalidTransaction, ModuleError, SignedExtension,
+	ValidTransaction, ValidateClaim,
 };
-use frame_support::dispatch::DispatchInfo;
+use frame_support::dispatch::{DispatchError, DispatchInfo};
 use frame_support::{assert_err, assert_noop, assert_ok};
 use hex_literal::hex;
 use sp_std::marker::PhantomData;
 
+/// convert an Error to a custom InvalidTransaction with the inner code being the error
+/// number.
+pub fn error_to_invalid<T: Config>(error: Error<T>) -> InvalidTransaction {
```

Comment:
what is the reason for duplicating this logic? We have namely the same function in `lib.rs`.

If no specific reason, then we could just import the one from `lib.rs`